composer.json

```diff
@@ -87,6 +87,11 @@
       "web/app/themes/{$name}/": ["type:wordpress-theme"]
     },
     "wordpress-install-dir": "web/wp",
+    "build-env": {
+      "install-cms": [
+        "wp option update permalink_structure '/%postname%/'"
```

Comment:
do we not need to run the `"wp core install --title={site-name} --url={site-url} --admin_user={account-name} --admin_email={account-mail} --admin_password={account-pass}",` command first?